pkg/alertmanager/amcfg.go

```diff
@@ -936,7 +1025,11 @@ func (c *alertmanagerConfig) sanitize(amVersion semver.Version, logger log.Logge
 
 	if len(c.MuteTimeIntervals) > 0 {
 		if muteTimeIntervalsAllowed := amVersion.GTE(semver.MustParse("0.22.0")); !muteTimeIntervalsAllowed {
-			return errors.Errorf(`mute_time_intervals is supported in Alertmanager >= 0.22.0 only (mute_time_intervals=%v)`, c.MuteTimeIntervals)
+			level.Warn(logger).Log(
+				"msg", `mute_time_intervals is supported in Alertmanager >= 0.22.0 only`,
+				"mute_time_intervals", c.MuteTimeIntervals,
+				"action", "dropping config",
+			)
```

Comment:
Will it be logged later? this is the sanitise so the last part of the "conversion pipeline" I believe unless I've missed something? This should now potentially only occur via the raw config I think, and instead of an error its a drop.

Correct with the reset, that was an oversight, thanks.

Reply:
I believe it will be logged for any route that will reference a mute time interval?

Reply:
ah yes, good catch, thanks